Userland/DevTools/UserspaceEmulator/SoftFPU.cpp

```diff
@@ -26,6 +26,15 @@
 
 namespace UserspaceEmulator {
 
+template<typename T>
```

Comment:
(In commit message. complience => compliance)

Reply:
fixed